modules/efk-logging-deploying-about.adoc

```diff
@@ -0,0 +1,220 @@
+          cpu:
+          memory:
+        requests:
+          cpu:
+          memory:
+     type: kibana
+  curation:
+    curator:
+      resources:
+        limits:
+          memory: 200Mi
+        requests:
+          cpu: 200m
+          memory: 200Mi
+      type: "curator"
+----
+
+Elasticsearch storage::
+You can configure a persistent storage class and size for the Elasticsearch cluster using the `storageClass` `name` and `size` parameters. The Cluster Logging Operator creates a `PersistentVolumeClaim` for each data node in the Elasticsearch cluster based on these parameters.  
+
+----
+  spec:
+    logStore:
+      type: "elasticsearch"
+      elasticsearch:
+        nodeCount: 3
+        storage:
+          storageClass: 
+            name: "gp2"
+            size: "200G"
+----
+
+This example specifies each data node in the cluster will be bound to a `PersistentVolumeClaim` that 
+requests "200G" of "gp2" storage.  Additionally, each primary shard will be backed by a single replica.
+
+[NOTE]
+====
+Omitting the `storage` block results in a deployment that includes ephemeral storage only.
+
+----
+  spec:
+    logStore:
+      type: "elasticsearch"
+      elasticsearch:
+        nodeCount: 3
+        storage: {}
+----
+====
+
+Elasticsearch replication policy::
+You can set the policy that defines how Elasticsearch shards are replicated across data nodes in the cluster:
+
+* `FullRedundancy`. The shards for each index are fully replicated to every data node.
+* `MultipleRedundancy`. The shards for each index are spread over half of the data nodes.
+* `SingleRedundancy`. A single copy of each shard. Logs are always available and recoverable as long as at least two data nodes exist.
+* `ZeroRedundancy`. No copies of any shards.  Logs may be unavailable (or lost) in the event a node is down or fails.
+
+Log collectors::
+You can select which log collector is deployed as a Daemonset to each node in the {product-title} cluster, either: 
+ 
+* Fluentd - The default log collector based on Fluentd. 
+* Rsyslog - Alternate log collector supported as **Tech Preview** only.
+
+----
+  spec:
+    collection:
+      logs:
+        fluentd:
+          resources:
+            limits:
+              cpu:
+              memory:
+            requests:
+              cpu:
+              memory:
+        type: "fluentd"
+----
+
+Curator schedule::
+You specify the schedule for Curator in the [cron format](https://en.wikipedia.org/wiki/Cron).
+
+----
+  spec:
+    curation:
+    type: "curator"
+    resources:
+    curator:
+      schedule: "30 3 * * *"
+----
+
+[id="efk-logging-deploy-about-sample-{context}"]
+== Sample modified Cluster Logging Custom Resource
+
+The following is an example of a Cluster Logging Custom Resource modified using the options previously described.
+
+.Sample modified Cluster Logging Custom Resource
+----
+apiVersion: "logging.openshift.io/v1alpha1"
+kind: "ClusterLogging"
+metadata:
+  name: "customresourcefluentd"
```

Comment:
we only support one named 'instance'.   We should explicitly mention this.  We did this to ensure there is only ever one instance of cluster logging for your OKD cluster.